### PR TITLE
MSCO-12: Display dropdown menu below each table header

### PR DESCRIPTION
**Pull request by pcandia**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -330,8 +330,11 @@ define(function(require) {
 						event.stopPropagation();
 
 						var $this = $(this),
-							$dropdown = $this.parents('th').find('.dropdown-menu-wrapper'),
-							$allDropdowns = container.find('#tasks_review_table .dropdown-menu-wrapper');
+							$dropdown = $this.parents('.column-data').find('.dropdown-menu-wrapper'),
+							$allDropdowns = container.find('#tasks_review_table .dropdown-menu-wrapper'),
+							minHeight = !$dropdown.hasClass('show')
+								? 20 + parseFloat($dropdown.css('height')) + parseFloat(container.find('.footable-header').css('height')) + 'px'
+								: 'unset';
```
Comment (joristirado) [inline app.js]: Here we can take advantage of `jquery#height` to compute the element's height. Also, what about storing those in variables to make it easier to understand what we're doing?

We can use `jquery#css` to un\set values as well, so no need to append `'px'` to our minHeight or use `unset`.
```suggestion
							headerHeight = parseFloat(container.find('.footable-header').height()),
							dropdownHeight = parseFloat($dropdown.height()),
							padding = 20,
							minHeight = $dropdown.hasClass('show')
								? ''
								: headerHeight + padding + dropdownHeight;
```

File: app.js [hunk 2]
```diff
@@ -342,6 +345,10 @@ define(function(require) {
 						$dropdown
 							.toggleClass('show');
 
+						container
+							.find('.review-table-wrapper')
+								.attr('style', 'min-height: ' + minHeight);
```
Comment (joristirado) [inline app.js]:
```suggestion
								.css('minHeight', minHeight);
```

File: app.js (hunk 1 above)
Comment (pcandia) [reply app.js]: Great advice, thank you!

File: app.js (hunk 2 above)
Comment (joristirado) [reply app.js]: There are two other instance of this in your changes, please update them as well to use `jquery#css` (you can unset properties by passing an empty string).

**Review by joristirado — APPROVED** [review]
Looks good to me!

**Pull request merged**